Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -297,7 +153,7 @@ - (UIFont *)textMessageFont
     OWSAssert(DisplayableText.kMaxJumbomojiCount == 5);
 
     CGFloat basePointSize = [UIFont ows_dynamicTypeBodyFont].pointSize;
-    switch (self.displayableText.jumbomojiCount) {
+    switch (self.displayableBodyText.jumbomojiCount) {
```

Comment:
As we move towards quoted replies, I've been trying to distinguish the "body" message/media from "quoted" message/media